src/Command/TwoStepVerificationCommand.php

```diff
@@ -16,34 +16,23 @@
 use FOS\UserBundle\Model\UserManagerInterface;
 use Sonata\UserBundle\GoogleAuthenticator\Helper;
 use Sonata\UserBundle\Model\UserInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
- */
-class TwoStepVerificationCommand extends ContainerAwareCommand
+class TwoStepVerificationCommand extends Command
```

Comment:
```suggestion
final class TwoStepVerificationCommand extends Command
```

Reply:
why this class can't be extended?

Reply:
@phansys I've applied your suggestion, But could you explain why add "final" here?

Reply:
For next major, we're trying to close the API by adding as many `final` keyword as possible.
This will allow us easier change to our base code.

Reply:
ohhhh thank you for explain :smiley: